src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,221 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, filterTypes, findTypes, omitProps } from '../../util/component-types';
+import * as reducers from './Sidebar.reducers';
+import SplitVertical from '../SplitVertical/SplitVertical';
+import ChevronIcon  from '../Icon/ChevronIcon/ChevronIcon';
+import GripperVerticalIcon  from '../Icon/GripperVerticalIcon/GripperVerticalIcon';
+
+const cx = lucidClassNames.bind('&-Sidebar');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	number,
+	string,
+	oneOf,
+	oneOfType,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"]}
+ *
+ * `Sidebar` renders a collapsible, resizeable side bar panel next to primary content.
+ */
+const Sidebar = createClass({
+	displayName: 'Sidebar',
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element. Value is run through the `classnames` library.
+		 */
+		className: string,
+		/**
+		 * Direct children must be types {Sidebar.Primary, Sidebar.Bar, Sidebar.Title}.
+		 * All content is composed as children of these respective elements.
+		 */
+		children: node,
+		/**
+		 * Sets the starting width of the Bar.
+		 */
+		width: oneOfType([number, string]),
+		/**
+		 * Force the Sidebar to be expanded or collapsed.
+		 */
+		isExpanded: bool,
+		/**
+		 * Allows animated expand and collapse behavior.
+		 */
+		isAnimated: bool,
+		/**
+		 * Render the Sidebar to the left or right of primary content.
+		 */
+		position: oneOf(['left', 'right']),
+		/**
+		 * Disable user resizing of the Sidebar.
+		 */
+		isResizeDisabled: bool,
+		/**
+		 * Set the title of the Sidebar. (alias for `Title` and `Sidebar.Title`)
```

Comment:
we haven't done it anywhere else because we never supported alias propNames until recently. I kept the upperCase propName to be in line with our other components, but i actually wanted to encourage consumers to use the prop in this case, which is why i also defined the lowercase propName.